README.md

```diff
@@ -35,6 +35,13 @@ Navigate to `http://localhost:3000` and log in:
 - To log in as a service provider user, use HMPPS Auth dev credentials e.g. `AUTH_ADM/password123456`
 - To log in as a probation practitioner user, use [Community API dev credentials](https://github.com/ministryofjustice/community-api/blob/main/src/main/resources/schema.ldif) e.g. `bernard.beaks/secret`.
 
+### Mocking out the interventions service
+
+In development mode, you might want to selectively mock out some calls to the
+interventions service. For example, for endpoints that have not yet been built.
+
+You can do this by configuring mocks in [`mocks.ts`](mocks.ts).
```

Comment:
Yeah, I think this is a good idea for now, as it'd be a pain to set up locally on everyone's machine. As long as we remember to remove them!

Reply:
👍